internal/clientconn/conn.go

```diff
@@ -167,7 +167,7 @@ func (c *conn) run(ctx context.Context) (err error) {
 
 	defer func() {
 		if p := recover(); p != nil {
-			c.l.LogAttrs(ctx, logging.LevelDPanic, fmt.Sprint("Failed to run", p), logging.Error(err))
+			c.l.LogAttrs(ctx, logging.LevelDPanic, fmt.Sprint("Failed to run: ", p), logging.Error(err))
```

Comment:
If that PR intends to sync changes with v2 (as the branch name indicates), then this line looks different there

Reply:
Yeah let's make the same.